index.bs

```diff
@@ -12421,6 +12417,42 @@ object is associated with.
     1.  Return |instance|.
 </div>
 
+<div algorithm>
+  To <dfn>define the global property references</dfn> on |target|, given [=Realm=] |realm|,
+  perform the following steps:
+
+    1.  [=list/iterate|For every=] non-callback [=interface=] |interface| that is [=exposed=] in
+        |realm|:
+        1.  If |interface| is not declared with the [{{NoInterfaceObject}}] or
+            [{{LegacyNamespace}}] [=extended attributes=], then:
+            1.  Let |id| be |interface|'s [=identifier=].
+            1.  Let |interfaceObject| be the result of [=create an interface object|creating
+                an interface object=] for |interface| with |id| in |realm|.
+            1.  Perform [=!=] [$CreateMethodProperty$](|target|, |id|, |interfaceObject|).
+            1.  If the |interface| is declared with a [{{LegacyWindowAlias}}] [=extended attribute=],
+                and |target| implements the {{Window}} [=interface=], then:
+                1.  [=list/iterate|For every=] [=LegacyWindowAlias identifier|identifier=] |id| in
+                    [{{LegacyWindowAlias}}]'s [=LegacyWindowAlias identifier|identifiers=]:
+                    1.  Perform [=!=] [$CreateMethodProperty$](|target|, |id|, |interfaceObject|).
+        1.  If the |interface| is declared with a [{{NamedConstructor}}] [=extended attribute=],
+            then:
+            1.  [=list/iterate|For every=] [=NamedConstructor identifier=] |id| on |interface|:
```

Comment:
Nit: above it's "For every id in [LegacyWindowAlias]'s identifiers"; here it's "For ever NamedConstructor identifier id". I like the first one more.